### PR TITLE
Add fadeOut for remove method

### PR DESCRIPTION
**Pull request by RomanBurunkov**
This adds option 'fade' for remove method, which fadeOut an item before removing it from list.

### CODE REVIEW

**Comment by RamonSmit** [comment]
LGTM, @pjona what do you think? :-)

**Comment by pjona** [comment]
@RomanBurunkov @RamonSmit maybe better solution is to give user some callback, so he can also use for example [`slideUp()`](https://api.jquery.com/slideUp/) with his duration time. What you think about it?

**Comment by pjona** [comment]
@RomanBurunkov I can help you with it, but I need more time to prepare some example.

**Comment by RamonSmit** [comment]
@pjona didn't think of that before. But that sounds like a good idea :+1:

**Comment by RomanBurunkov** [comment]
@pjona, do you mean adding callback as a parameter during invoke remove method?

But we also can add some additional methods for hiding with my approach and also add time var...

Adding callback can be more comprehensive, but adding several hiding types and time variable is simplier to use: 

```
$('#nest').nestable('remove', 'fade', 300);
$('#nest').nestable('remove', 'slide', 300);
```

or

```
$('#nest').nestable('remove', function(item){
   ....
});
```

File: jquery.nestable.js [hunk 1]
```diff
@@ -243,31 +242,42 @@
...
+                var buttonsSelector = '[data-action="expand"], [data-action="collapse"]';
+                $(el).find(buttonsSelector).each(function() {
+                    var siblings = $(this).siblings('.' + opts.listClass);
+                    if (siblings.length === 0) {
+                        $(this).remove();
+                    }
+                });                
+            }
 
-        _getItemById: function(itemId) {
```
Comment (pjona) [inline jquery.nestable.js]: I think we need this method :)

**Comment by pjona** [comment]
@RomanBurunkov I think solution with 3 parameters fits enough:

```
$('#nest').nestable('remove', 'fade', 300);
$('#nest').nestable('remove', 'slide', 300);
```

File: jquery.nestable.js (hunk 1 above)
Comment (RamonSmit) [inline jquery.nestable.js]: Simple but elegant 👍

Comment (RomanBurunkov) [inline jquery.nestable.js]: It's here:

`
var opts = this.options,
      el   = this.el,
      item = this._getItemById(itemId)
`

Comment (RomanBurunkov) [inline jquery.nestable.js]: Agree =)
Will double check this.

Comment (pjona) [inline jquery.nestable.js]: I mean, I see you removed this method but we have two places where we still using it.

Comment (RamonSmit) [inline jquery.nestable.js]: Oh really, did not notice! My fault! Glad you're looking @pjona

Comment (RomanBurunkov) [inline jquery.nestable.js]: @pjona, That's looks like my fault(I've several versions of nestable on my PC, where I changed location for this method, so here I've cut them and forgot to paste :(.... ). I will add another commit to this PR, today.

**Pull request merged**